include/groonga/groonga.h

```diff
@@ -980,6 +981,19 @@ grn_obj_set_value(
 #define GRN_OBJ_REMOVE_DEPENDENT (0x01 << 0)
 #define GRN_OBJ_REMOVE_ENSURE    (0x01 << 1)
 
+/**
+ * \brief Remove the specified object.
+ *
+ *        This function frees the specified object (`obj`) from memory. If the
+ *        object is a persistent object, it also removes the associated files
+ *        from disk.
+ *
+ * \param ctx The context object.
+ * \param obj The target object to be removed.
+ *
+ * \return \ref GRN_SUCCESS on success, \ref GRN_UNKNOWN_ERROR on error, and
+ *         the appropriate \ref grn_rc for any other errors.
```

Comment:
```suggestion
 * \return \ref GRN_SUCCESS on success, the appropriate \ref grn_rc on error.
```

Reply:
fix: [ba94707](https://github.com/groonga/groonga/pull/1916/commits/ba947078247e67a52a4036534346eee02e9c4538) Thanks you so much mm.